backends-velox/src/main/scala/org/apache/spark/sql/VeloxColumnarRules.scala

```diff
@@ -24,16 +24,16 @@ import org.apache.spark.sql.execution._
 import org.apache.spark.sql.execution.adaptive.AdaptiveSparkPlanExec
 import org.apache.spark.sql.execution.command.DataWritingCommandExec
 import org.apache.spark.sql.execution.datasources.InsertIntoHadoopFsRelationCommand
-import org.apache.spark.sql.execution.datasources.velox.DwrfFileFormat
+import org.apache.spark.sql.execution.datasources.velox.VeloxParquetFileFormat
 
 object VeloxColumnarRules {
 
   case class OtherWritePostRule(session: SparkSession) extends Rule[SparkPlan] {
     override def apply(plan: SparkPlan): SparkPlan = plan match {
-      case rc@DataWritingCommandExec(cmd, ColumnarToRowExec(child)) =>
+      case rc@DataWritingCommandExec(cmd, GlutenColumnarToRowExec(child)) =>
         cmd match {
           case command: InsertIntoHadoopFsRelationCommand =>
-            if (command.fileFormat.isInstanceOf[DwrfFileFormat]) {
+            if (command.fileFormat.isInstanceOf[VeloxParquetFileFormat]) {
```

Comment:
Rename `VeloxParquetFileFormat` to `ParquetFileFormat`, since velox parquet has no difference with standard parquet

Reply:
Yes. The velox parquet is same with standard apache parquet. Here is to distinguish with ParquetFileFormat in [Datasource](https://github.com/apache/spark/blob/6d4ed13c465ae7f35753a1f2b67f78c690881a9e/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSource.scala#L669).